### PR TITLE
Enhancement/skale 1793 Update to new skale py version

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: tests/integration/preparation.py [hunk 1]
```diff
@@ -19,48 +19,41 @@
...
+from tools.configs.web3 import ABI_FILEPATH, ENDPOINT
+from tools.helper import get_local_wallet_filepath
 
 DIR_LOG = '/skale_node_data/log'
 DIR_ABI = '/skale_vol/contracts_info'
 TEST_EPOCH = 200
 TEST_DELTA = 100
-
-skale = init_skale()
+SKL_DEPOSIT = 100
```
Comment (dmytrotkk) [inline tests/integration/preparation.py:36]: we should request this value from the network

File: tests/integration/preparation.py [hunk 2]
```diff
@@ -154,16 +130,14 @@ def create_set_of_nodes(first_node_id, nodes_number):
...
-    print(reward_period, delta_period)
+    print(f'New times for SM: {reward_period}, {delta_period}')
 
-    wallet = {'address': account, 'private_key': pr_key}
-    res = skale.constants.set_periods(TEST_EPOCH, TEST_DELTA, wallet)
-    receipt = await_receipt(skale.web3, res['tx'], retries=30, timeout=6)
+    res = skale.constants.set_periods(TEST_EPOCH, TEST_DELTA)
+    receipt = wait_receipt(skale.web3, res['tx'], retries=30, timeout=6)
     print(receipt)
     print("-------------------------")
```
Comment (dmytrotkk) [inline tests/integration/preparation.py:142]: mb it's better to move this to constants?

**Review by dmytrotkk — APPROVED** [review]
does it work fine with skale.py 2.0?

**Comment by alexgex** [comment]
> does it work fine with skale.py 2.0?

yes)

File: tests/integration/preparation.py (hunk 2 above)
Comment (alexgex) [reply tests/integration/preparation.py:142]: oh, thanks, forgot to delete it (just a temporary debug output)

**Pull request merged**